### PR TITLE
MCLOUD-5511: Add Multi-website Support into Cloud Docker

### PR DESCRIPTION
**Pull request by shiftedreality**
## Purpose of this pull request

https://jira.corp.magento.com/browse/MCLOUD-5511

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/docker/docker-config.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by nuzil** [comment]
@shiftedreality can you please check tests?

File: src/cloud/docker/docker-multi-website.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - Setup
+  - Configuration
+  - Website
+---
+
+Magento Cloud Docker supports multi-website by using different subdomains.
+
+## Add subdomains
+
+1. Ensure the `php.ini` file included and pints to correct `magento-vars.php` configration
```
Comment (meker12) [inline src/cloud/docker/docker-multi-website.md:15]:
```suggestion
1. Ensure that the `php.ini` file includes the configuration for the `magento-vars.php` file:
```

File: src/cloud/docker/docker-multi-website.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    ```ini
+    auto_prepend_file = /app/magento-vars.php
+    ```
+
+Magento Cloud Docker read the `php.ini` file if it is included in your project.
+
+1. Each subdomain must be added to `/etc/hosts` configuration file
```
Comment (meker12) [inline src/cloud/docker/docker-multi-website.md:23]:
```suggestion
1. Add each subdomain to the `/etc/hosts` configuration file.
```

File: src/cloud/docker/docker-multi-website.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Add subdomains
+
+1. Ensure the `php.ini` file included and pints to correct `magento-vars.php` configration
+
+    ```ini
+    auto_prepend_file = /app/magento-vars.php
+    ```
+
+Magento Cloud Docker read the `php.ini` file if it is included in your project.
```
Comment (meker12) [inline src/cloud/docker/docker-multi-website.md:21]:
```suggestion
    ```ini
    auto_prepend_file = /app/magento-vars.php
    ```
    {:.bs-callout-info}
    {{ site.data.var.mcd-prod }} applies the configuration in the `php.ini` file if it is included in the root directory of your project.
```

File: src/cloud/docker/docker-multi-website.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+1. Each subdomain must be added to `/etc/hosts` configuration file
+
+    ```
+    127.0.0.1 magento2.docker
+    127.0.0.1 second.magento2.docker
+    ```
+
+{:.bs-callout-info}
+You can make next changes while your Cloud Docker is running.
+
```
Comment (meker12) [inline src/cloud/docker/docker-multi-website.md:32]:
```suggestion
    ```

    {:.bs-callout-info}
   You can make next changes while your Cloud Docker is running.

```

File: src/cloud/docker/docker-multi-website.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+1. Add specific stores and websites in Magento admin:
+
+   - [Set up websites, stores, and store views]({{site.baseurl}}/cloud/project/project-multi-sites.html#set-stores)
+
+1. Use `magento-vars.php` configuration to add Magento store and website codes:
```
Comment (meker12) [inline src/cloud/docker/docker-multi-website.md:37]:
```suggestion
1. Add the configuration for Magento store and website codes to the `magento-vars.php` file:
```

**Comment by meker12** [comment]
running tests

**Pull request merged**